scripts/count_opcodes.py

```diff
@@ -161,6 +163,7 @@ def report(self, source, filename, verbose, bias):
 
         for co in all_code_objects(code):
             counter[NCODEOBJS] += 1
+            counter[CO_CONSTS_SIZE] += len(co.co_consts)
```

Comment:
Same constants are shared. I want to measure it too.

```suggestion
        consts_ids = set()
        for co in all_code_objects(code):
            counter[NCODEOBJS] += 1
            if id(co.co_consts) not in consts_ids:
                consts_ids.add(id(co.co_consts))
                counter[CO_CONSTS_SIZE] += len(co.co_consts)
```

Reply:
Oh, this would avoid counting e.g. `(None,)` twice if the same object occurs as `co_consts` in two different code objects. It's true that we may be over-counting this.